sheet.go

```diff
@@ -821,11 +821,18 @@ func parseFormatPanesSet(formatSet string) (*formatPanes, error) {
 //    f.SetPanes("Sheet1", `{"freeze":false,"split":false}`)
 //
 func (f *File) SetPanes(sheet, panes string) error {
-	fs, _ := parseFormatPanesSet(panes)
 	ws, err := f.workSheetReader(sheet)
 	if err != nil {
 		return err
 	}
+
+	ws.setPanes(panes)
+	return nil
+}
+
+func (ws *xlsxWorksheet) setPanes(panes string) {
+	fs, _ := parseFormatPanesSet(panes)
```

Comment:
I have ignored the error of parsing the JSON, which is not a good design, add an error return for the `setPanes` for the future error return. Similar to the `NewStyle` function, I'll export the `formatPanes` structure and add param validation in the future.